src/deadline/blender_submitter/addons/deadline_cloud_blender_submitter/template_filling.py

```diff
@@ -49,7 +49,9 @@ class BlenderSubmitterUISettings:
 
     # Paths and files.
     project_path: str = field(default="")
-    output_path: str = field(default="")
```

Comment:
Does this need to be a breaking change by doing this rename?

Reply:
For visibility: we talked about this offline and the reasoning is that this rename is a breaking change for people building off this repository.

My preference for `directory` over `path` is purely semantic so we can keep the name as-is, as long as it's clear that it's the path to the directory :)